examples/python/debug/README.md

```diff
@@ -0,0 +1,43 @@
+# gRPC Python Debug Example
+
+This example demonstrate the usage of Channelz. For a better looking website, the [gdebug](https://github.com/grpc/grpc-experiments/tree/master/gdebug) uses gRPC-Web protocol and will serve all useful information in web pages.
```

Comment:
Nit: Textwrap at 80 characters?

Reply:
Re-wrapped the README to 80 characters per line.